tfs/testing.py

```diff
@@ -0,0 +1,48 @@
+"""
+Testing
+-------
+
+Testing functionalty for TfsDataFrames.
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from pandas._testing import assert_dict_equal
+from pandas.testing import assert_frame_equal
+
+if TYPE_CHECKING:
+    from tfs.frame import TfsDataFrame
+
+
+# ----- Helpers ----- #
+
+
+def assert_tfs_frame_equal(
+    df1: TfsDataFrame, df2: TfsDataFrame, compare_keys: bool = True, **kwargs
+):
+    """
+    Compare two `TfsDataFrame` objects, this is mostly intended
+    for unit tests. Comparison is done on both the contents of
+    the headers dictionaries (with `pandas`'s `assert_dict_equal`)
+    as well as the data itself (with `pandas`'s `assert_frame_equal`).
+
+    Args:
+        df1 (TfsDataFrame): The first `TfsDataFrame` to compare.
+        df2 (TfsDataFrame): The second `TfsDataFrame` to compare.
+        compare_keys (bool): If `True`, conpares the headers keys
+            as well. Defaults to `True`.
```

Comment:
`conpares` typo. But also I would specify how this works. I have not really found documentation on this, but from my experience it looks like:
compare_keys=False -> checks only that common keys in the header dicts have the same values
compare_keys=True -> raises errors if there are different keys in the dict

Reply:
Yes as raised by @jgray-19 this needed more details. I am adding now.